crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -1844,6 +1844,31 @@ _.tt.tttt.t.tt.ttttt_
             -_-
 ENDMAP
 
+#### A dangerous temple to kiku
+NAME: beargit_kiku_chapel
+DEPTH: Swamp:2-
+TAGS: no_monster_gen
+KMONS: 1 = bunyip zombie w:10 / bunyip skeleton w:10 / bog body w:20 / death scarab w:1
+KMONS: 2 = patrolling juggernaut zombie w:20 / patrolling caustic shrike zombie w:3 / patrolling shard shrike zombie w:3 / patrolling spectral juggernaut w:5 / patrolling spectral caustic shrike w:1
+KMONS: 4 = patrolling lich w:1 / patrolling fenstrider_witch; quick blade ego:pain w:1
+KMONS: 5 = ghost crab w:10 / death drake w:1 / ghoul w:1
+KFEAT: J = altar_kikubaaqudgha
+KITEM: K = shadow dragon skeleton
+KITEM: L = frost giant skeleton
+FTILE: . = floor_sandstone
+ITEM: randbook disc:necromancy slevels:12
+SUBST: x = xxxxxxxxxxxxx..WWWwwt.
+TILE: c = wall_brick_vines, x = wall_brick_vines
```

Comment:
Don't retile stone and rock to the same tile. Not sure what your target aesthetic is so I don't have a good suggestion for replacement, other than we typically leave stone as-is in swamp.

Reply:
is the issue the SUBST or the TILE?

Reply:
Fixed.